src/store/mod.rs

```diff
@@ -216,6 +194,67 @@ pub mod tests {
         }
         Ok(())
     }
+
+    #[cfg(feature = "snap")]
+    #[cfg(feature = "lz4_flex")]
```

Comment:
I'm confused by the feature name in this file.
Aren't they supposed to be called respectively `{snappy, lz4, brotli}_compression` .